robottelo/config/base.py

```diff
@@ -879,7 +879,7 @@ def _validate_robottelo_settings(self):
         """Validate Robottelo's general settings."""
         validation_errors = []
         browsers = ('selenium', 'docker', 'saucelabs')
-        webdrivers = ('chrome', 'firefox', 'ie', 'phantomjs', 'remote')
+        webdrivers = ('chrome', 'edge', 'firefox', 'ie', 'phantomjs', 'remote')
```

Comment:
Still thinking about removing ie class and etc to keep only one MS browser supported